fhir2dataset/url_builder.py

```diff
@@ -62,14 +61,18 @@ def _get_url_params(self):
                 for search_param, values in infos_search_param.items():
                     # add assert search_param in CapabilityStatement
                     value = f"{values['prefix'] or ''}{values['value']}"
-                    url_temp = f"{f'{url_temp}&' if url_temp else ''}{to_resource or ''}{search_param}={value}"
+                    url_temp = (
+                        f"{f'{url_temp}&' if url_temp else ''}{to_resource or ''}"
```

Comment:
same

Reply:
I understand the problem raised in the previous comment. However I'm not sure I understand the problem at this level (it's the construction of a portion of the url that does not contain a slash)